app/lib/common/widgets/app.dart

```diff
@@ -1,17 +1,16 @@
-import 'package:flutter/material.dart';
+import 'package:flutter/material.dart' hide MetaData;
```

Comment:
```suggestion
import 'package:flutter/material.dart' hide MetaData;
```

Hmm, maybe if we need to do this we should think about renaming the class to something else, maybe UserMetaData